src/bosh-director/spec/unit/deployment_plan/persistent_disk_collection_spec.rb

```diff
@@ -349,6 +349,44 @@ module DeploymentPlan
           end
         end
       end
+
+      describe '#is_bigger_than?' do
+        let(:new_disk) { new_disk = PersistentDiskCollection::PersistentDisk.new('', {}, 10) }
+
+        context 'when other thing is not a disk' do
+          it 'raises an error' do
+            old_disk = {}
+
+            expect {
+              new_disk.is_bigger_than?(old_disk)
+            }.to raise_error(/Cannot compare persistent disk size to anything that is not a persistent disk/)
+          end
+        end
+
+        context 'when the size is bigger' do
+          it 'is true' do
+            old_disk = PersistentDiskCollection::PersistentDisk.new('', {}, 5)
+
+            expect(new_disk.is_bigger_than?(old_disk)).to eq(true)
+          end
+        end
+
+        context 'when the size is equal' do
+          it 'is true' do
```

Comment:
Does it make sense to return true here? Haven't followed the code all the way through, but is it desirable to do an IAAS resize if the disks are equal in size? Or does this get caught somewhere else and that would never happen?

Reply:
Well, we have those two conditions in [bosh/director/disk_manager.rb:91-92](https://github.com/cloudfoundry/bosh/pull/2318/files#diff-310819dc5baeabbb6d5bc86cb5afa6e273a95556306fc1cb88fe680ee9489230R91-R92):
```
new_disk.size_diff_only?(old_disk) &&
new_disk.is_bigger_than?(old_disk)
```
So indeed, we capture the two conditions (difference in size & bigger size) separately.

Reply:
It's true that we capture that case somewhere else. Just from a unit test point of view it does not make sense to return true. Since the disk is not bigger when the size is equal.

Reply:
Will merged as is and apply https://github.com/cloudfoundry/bosh/commit/b4a9c20bca63d5b06e25e1ad40d195f4f9a39801

Reply:
All right, make sense!